NEWS.rst

```diff
@@ -6,6 +6,11 @@ New features
 
 * #22849: Implemented support for signed SAML AuthN Requests.
 
+Bug fixes
+---------
+
+* #24067: Fixed being able to edit root organisational units
```

Comment:
“Allowed editing root organisational units.”?

Strictly speaking, I wouldn't say this was a bug, but rather a missing feature.

Reply:
We never mentioned _not_ being able to edit root organisational units, and the error message wasn't particularly helpful - so I think this qualifies as a bug.